### PR TITLE
changes made

### PR DESCRIPTION
**Pull request by leren123**
# Description

Modified file:
mlfinlab/docs/source/implementations/clustering.rst

New files:
mlfinlab/docs/source/implementations/labeling_images/Euclidean_distance_formula.PNG
mlfinlab/docs/source/implementations/labeling_images/proper_distance_matrix_formula.PNG
mlfinlab/docs/source/implementations/labeling_images/quality_distance_formula.PNG
mlfinlab/docs/source/implementations/labeling_images/silhouette_score_formula.PNG

Thank you

### CODE REVIEW

File: docs/source/implementations/clustering.rst [hunk 1]
```diff
@@ -9,7 +9,109 @@ This implementation is based on 'Detection of False Investment Strategies using
 https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3167017
 
 
+Implementation
```
Comment (Jackal08) [inline docs/source/implementations/clustering.rst]: Why did you save the images into the labeling folder? These are not labeling images. Please add them to the correct folder, if none exists please create one.

File: mlfinlab/docs/source/implementations/clustering.rst [hunk 2]
```diff
@@ -0,0 +1,117 @@
+.. _implementations-clustering:
```
Comment (Jackal08) [inline mlfinlab/docs/source/implementations/clustering.rst]: Why are there two files called clustering? I'm confused, how was it even possible to commit 2 files in the same dir with the same name?

Comment (Jackal08) [reply mlfinlab/docs/source/implementations/clustering.rst]: Hmm please double check. There are 8 files here and 4 of them are duplicates but not quite.

File: docs/source/implementations/clustering.rst (hunk 1 above)
Comment (leren123) [reply docs/source/implementations/clustering.rst:12]: hello,

I have made a new folder called 'formula_images' and put the necessary PNG files to the folder and deleted the same PNG files from the folder of 'labeling_images'  - thanks

**Pull request closed without merging**